styleguide/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ama-style-guide",
-  "version": "2.25.3",
+  "version": "2.25.4",
```

Comment:
Super minor @designerneil , can you make this 2.26.0?
```suggestion
  "version": "2.26.0",
```

Reply:
why are we jumping to 2.26.0?

Reply:
We're in Sprint 26 right?